README.md

```diff
@@ -1,24 +1,41 @@
-# README
+# ClubHouse
+This project is a Clubhouse app. The idea is to create an app where the members of the Club can publish posts anonymously. Everyone outside the club should be able to see those posts but not the name of the author unless they are members of the Club.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## App Home Page
+![](app/assets/images/screenshot.png)
 
-Things you may want to cover:
+## Getting Started :
+- Clone the repository: `git clone` https://github.com/NoTengoBattery/club-house.git
+- Move inside the club-house folder: `cd club-house`
+- Run the server: `rails s` or `rails server`
+- Run: `rails db:create` and `rails db:migrate`.
```

Comment:
Run rails db:create and rails db:migrate should come before Run the server: rails s or rails server.